sentry-android-core/src/main/java/io/sentry/android/core/ScreenshotEventProcessor.java

```diff
@@ -51,10 +54,86 @@ public ScreenshotEventProcessor(
     application.registerActivityLifecycleCallbacks(this);
   }
 
+  public static ScreenshotEventProcessor createInstance(
```

Comment:
```suggestion
  public static @NotNull ScreenshotEventProcessor getInstance(
```

H: Always use the nullability annotations, better Kotlin interoperability.
M: Stick to the convention of `getInstance` naming.